build-llvm.sh

```diff
@@ -322,5 +322,6 @@ cmake \
 
 cmake --build . ${CORES:+-j${CORES}}
 cmake --install . --strip
+cmake --install . --strip --component clang-scan-deps
```

Comment:
This isn't necessary here, the plain regular install above does install the binary already. The only thing that has to be done is to update `strip-llvm.sh` to retain it.

Reply:
I updated the PR with the relevant change to `strip-llvm.sh` and removed this change for `build-llvm.sh` (and adjusted the commit message accordingly). With that, this PR looks good, and I'll go ahead and merge it soon if you don't mind the changes.

Reply:
i dont mind, but i was certain that i need the extra install step